### PR TITLE
Patch for Learning Outcomes

### PR DESCRIPTION
**Pull request by mfranz1**
This PR to patch an error when creating learning outcomes. Relates to [8592](https://app.shortcut.com/clarkcan/story/8592/none-used-verb-being-accepted-and-crashing-outcomes-service) and [8804](https://app.shortcut.com/clarkcan/story/8804/clark-dashboard-not-showing-learning-objects-on-client)

### CODE REVIEW

File: src/app/onion/learning-object-builder/builder-store.service.ts [hunk 1]
```diff
@@ -1053,7 +1060,13 @@ export class BuilderStore {
       .then(() => {
         this.serviceInteraction$.next(false);
       })
-      .catch(e => this.handleServiceError(e, BUILDER_ERRORS.UPDATE_OUTCOME));
+      .catch(e => {
+        if (e.status !== 406) {
+          this.handleServiceError(e, BUILDER_ERRORS.UPDATE_OUTCOME)
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/builder-store.service.ts:1065]: Couldn't we add the 406 to this function?

File: src/entity/learning-outcome/learning-outcome.ts [hunk 2]
```diff
@@ -31,14 +31,15 @@ export class LearningOutcome {
...
-    if (bloom && levels.includes(bloom.toLowerCase())) {
-      this._bloom = bloom.toLowerCase();
-    } else {
-      throw new EntityError(
-        LEARNING_OUTCOME_ERROR_MESSAGES.INVALID_BLOOM(bloom),
-        'bloom',
-      );
-    }
+    this._bloom = bloom.toLowerCase();
+    // This logic works; it just belongs in a service
```
Comment (pzalep1) [inline src/entity/learning-outcome/learning-outcome.ts:35]: Are we going to uncomment this or delete it?

**Review by pzalep1 — CHANGES_REQUESTED** [review]

File: src/app/onion/learning-object-builder/builder-store.service.ts (hunk 1 above)
Comment (JosueMolinaMorales) [reply src/app/onion/learning-object-builder/builder-store.service.ts:1065]: I agree

File: src/entity/learning-outcome/learning-outcome.ts (hunk 2 above)
Comment (JosueMolinaMorales) [reply src/entity/learning-outcome/learning-outcome.ts:35]: ^

File: src/entity/learning-outcome/learning-outcome.ts [hunk 3]
```diff
@@ -51,20 +52,21 @@ export class LearningOutcome {
...
+    //   (taxonomy.taxons[this.bloom] as { verbs: string[] }).verbs.includes(
+    //     verb.toLowerCase(),
+    //   )
+    // ) {
+    // } else {
+    //   throw new EntityError(
+    //     LEARNING_OUTCOME_ERROR_MESSAGES.INVALID_VERB(this.bloom, verb),
+    //     'verb',
+    //   );
+    // }
```
Comment (JosueMolinaMorales) [inline src/entity/learning-outcome/learning-outcome.ts:69]: same here

**Review by JosueMolinaMorales — COMMENTED** [review]
Couple things and Paige's comments

**Review by EVogel1999 — APPROVED** [review]

**Review by StarChaser44 — DISMISSED** [review]

**Review by JosueMolinaMorales — DISMISSED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Review by pzalep1 — DISMISSED** [review]

**Review by EVogel1999 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**